src/imdb_detector.py

```diff
@@ -64,12 +65,12 @@ def get_column_specific_label_mapping(self) -> dict:
             "cast_id": set_all_labels_to_ocr,
             "cast_person_id": set_all_labels_to_ocr,
             "cast_movie_id": set_all_labels_to_ocr,
-            "cast_person_role_id": differentiate_errors_in_string_column,
+            "cast_person_role_id": set_all_labels_to_ocr,
             "cast_note": differentiate_errors_in_string_column,
             "cast_nr_order": set_all_labels_to_ocr,
             "cast_role_id": set_all_labels_to_ocr,
             "person_id": set_all_labels_to_ocr,
-            "person_movie_id": differentiate_errors_in_string_column,
+            "person_movie_id": set_all_labels_to_ocr,
```

Comment:
Did you check these "manually" with the unique values script that all invalid unique values are OCRs? If so, I think its good to put a comment so that we know that we never have to check them again. In the PR #35 I standardized these comments to 
> Manual check -> all OCRs

Or for ID columns:
> IDs have no typos -> OCR